src/Library/demos/Drawing Area/main.blp

```diff
@@ -0,0 +1,31 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Drawing Area";
+  description: _("Connection between control and drawing");
```

Comment:
the description isn't about the library entry itself but rather what is being presented, here it's "Drawing Area"

```suggestion
  description: _("Programmatically draw onto a surface.");
```